README.md

```diff
@@ -76,11 +81,19 @@ the actual driver's name.
           args:
             - "--csi-address=/csi/csi.sock"
             - "--kubelet-registration-path=/var/lib/kubelet/plugins/<drivername.example.com>/csi.sock"
+            - "--health-port=9809"
           volumeMounts:
             - name: plugin-dir
               mountPath: /csi
             - name: registration-dir
               mountPath: /registration
+          ports:
+            - containerPort: 9809
+              name: healthz
+          livenessProbe:
+            httpGet:
```

Comment:
Can you also test this out in a driver Pod where `hostNetwork: true`? I'm not sure if anything special needs to be done to configure the livenessProbe in that scenario

Reply:
Just tested on a GCE node, didn't encounter any issues, was able to see the health checks pass (and fail when I removed the registration port).